releases/fluentd-splunk-logs.yaml

```
@@ -0,0 +1,52 @@
+releases:
+
+#######################################################################################
+## Fluentd to Splunk logs                                                            ##
+## Forward logs to Splunk with fluentd                                               ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/splunk/splunk-connect-for-kubernetes/blob/master/helm-chart/splunk-kubernetes-logging/values.yaml
+#
+
+- name: "fluentd-splunk-logs"
+  namespace: "kube-system"
+  labels:
+    chart: "splunk-kubernetes-logging"
+    repo: "splunk"
+    component: "splunk"
+    namespace: "kube-system"
+    default: "false"
+  chart: "https://github.com/splunk/splunk-connect-for-kubernetes/releases/download/v1.0.1/splunk-kubernetes-logging-1.0.1.tgz"
+  wait: true
+  values:
+  - splunk:
+      # Configurations for HEC (HTTP Event Collector)
+      hec:
+        # host is required and should be provided by user
+        host: '{{ requiredEnv "FLUENTD_SPLUNK_HOST" }}'
+        # port to HEC, optional, default 8088
+        port: '{{ env "FLUENTD_SPLUNK_PORT" | default "8088" }}'
+        # token is required and should be provided by user
+        token: '{{ requiredEnv "FLUENTD_SPLUNK_TOKEN" }}'
+        # protocol has two options: "http" and "https", default is "https"
+        protocol: '{{ env "FLUENTD_SPLUNK_PROTOCOL" | default "https"}}'
+        # indexName tells which index to use, this is optional. If it's not present, will use the default one configured in HEC.
+        indexName: '{{ requiredEnv "FLUENTD_SPLUNK_INDEX_NAME" }}'
```

```suggestion
        indexName: '{{ env "FLUENTD_SPLUNK_INDEX_NAME" }}'
```

I don't think we can use `requiredEnv` since this gets really annoying if you're trying to deploy another release that's *totally unrelated*, you still need to define this unused env. I think it's nice in principle, poor in practice.